contents/docs/cdp/batch-exports/index.md

```diff
@@ -163,3 +163,11 @@ sequenceDiagram
 If no data is found for a particular batch period, then the batch export run will immediately succeed. No data to export is not considered a failure, as it is expected that there may be times of low or no volume of events coming into PostHog, and it is particularly relevant when filtering the batch export to only export very low volume events.
 
 > **Note:** This could mean that underlying configuration issues are not surfaced early, as we do not attempt to connect to a destination if there is nothing to export. It is recommended to confirm there is data to export to ensure a batch export has been configured correctly. This can be done by checking batch export debug logs as they will display how many rows are exported. So, if no debug logs appear, it means there is no data to export, so we haven't had a chance to validate the batch export configuration.
+
+### Why do I see duplicate data when running a backfill?
+
+A backfill does not check if the data already exists in the destination. Doing so would negatively impact performance of each batch export run, and potentially require more permissions on the destination. Moreover, we can never be sure if the data was moved somewhere else. Instead, we assume that users who start a backfill want all their historic data, which means that multiple batch export runs over the same time period will produce duplicates.
+
+One other case in which you may see duplicate events is when the batch export runs transition from backfills to regular runs (for example, if you create a backfill to export all your historical data, and then you enable the batch export to start exporting data from the current time). The reason for this is as follows: If the batch export is for the events model and the interval start is older than 6 days ago then the event data will be queried based on the `timestamp` column (this is for performance reasons). In all other cases, we use an internal `inserted_at` column, which is based on when the event was ingested into PostHog's ClickHouse database (this prevents issues related to replication lag).
```

Comment:
I think this is confusing more than explaining. But also not sure exactly how I would prefer to write it.

Reply:
I meant the second paragraph, my review highlighted both, but I think the first one is fine.

Reply:
I've simplified this now for clarity. Let me know what you think